routes/exchange_test.go

```diff
@@ -567,9 +567,9 @@ func TestAPI(t *testing.T) {
 	var secondBlockTxn *lib.MsgDeSoTxn
 	{
 		blockHash := apiServer.blockchain.BestChain()[1].Hash
-		blockLookup, err := lib.GetBlock(blockHash, apiServer.blockchain.DB())
```

Comment:
This code produces a compiler error. Assuming this test was just unmaintained after a migration

Reply:
correct